docs/reference/ml/df-analytics/apis/delete-dfanalytics.asciidoc

```diff
@@ -21,8 +21,12 @@ experimental[]
 [[ml-delete-dfanalytics-prereq]]
 ==== {api-prereq-title}
 
-* You must have `machine_learning_admin` built-in role to use this API. For more
-information, see <<security-privileges>> and <<built-in-roles>>.
+Required build-in roles:
```

Comment:
I think it would be clearer to have a more complete phrase, without compromising the simplicity of the bulleted list. For example:
```suggestion
If the {es} {security-features} are enabled, you must have the following built-in roles or equivalent privileges:
```

Reply:
@lcawl Thanks, I accepted the change and applied it to the rest of the files in the following manner:

**If the API requires only built-in roles:**
If the {es} {security-features} are enabled, you must have the following built-in roles or equivalent privileges:

* (bulleted list of built-in roles)

**If the API requires only privileges:**
If the {es} {security-features} are enabled, you must have the following privileges:

* (bulleted list of privileges)

**If the API requires both build-in roles and privileges:**
If the {es} {security-features} are enabled, you must have the following built-in roles and privileges:

* (bulleted list of build-in roles)

* (bulleted list of privileges)